### PR TITLE
framework/media: Add MediaPlayer.isPlaying API

### PR DESCRIPTION
**Pull request by junmin-kim**
isPlaying() checks whether MediaPlayer is playing

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [fb5152a52f4dfab7cbea2eea795e5fd52c694f6b] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:219: ERROR: [SPC_M_OPR] spaces required around that '==' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:141: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Comment by seinfra** [comment]
Target : [fb5152a52f4dfab7cbea2eea795e5fd52c694f6b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6af6b58186a8276d1c10a0e1678d7ac36cf6d646] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6af6b58186a8276d1c10a0e1678d7ac36cf6d646] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:219: ERROR: [SPC_M_OPR] spaces required around that '==' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:141: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

File: apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp [hunk 1]
```diff
@@ -478,6 +478,32 @@ static void utc_media_MediaPlayer_setVolume_n(void)
...
+	mp.unprepare();
+	mp.destroy();
+
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_media_MediaPlayer_isPlaying_n(void)
+{
+	media::MediaPlayer mp;
+
```
Comment (jsdosa) [inline apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp]: could you add one case~? after mp.create()
Substantially, that is a return to what was not mp create.

Comment (junmin-kim) [reply apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp]: I added `after mp.create()` test case

**Comment by seinfra** [comment]
Target : [98141930cfeae6c76cb408a60e63590987e9b527] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [98141930cfeae6c76cb408a60e63590987e9b527] - Code Rule Check Result:
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:198: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:209: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/include/media/MediaPlayer.h:219: ERROR: [SPC_M_OPR] spaces required around that '==' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/include/media/MediaPlayer.h:230: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:88: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxE)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:107: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:108: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxB)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:130: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:VxO)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:131: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:135: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:138: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:139: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '<' (ctx:VxV)
framework/src/media/MediaPlayerImpl.h:140: ERROR: [SPC_M_OPR] spaces required around that '>' (ctx:VxW)
framework/src/media/MediaPlayerImpl.h:141: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Pull request merged**